launch_testing/launch_testing/__init__.py

```diff
@@ -17,12 +17,14 @@ class InMemoryHandler(LineOutput):
         accordingly.
     @param filtered_prefixes: A list of prefixes that will be ignored. By default the output of
         RTI Connext will be ignored.
+    @param test_process: If True, then when its output is matched, this handler
+        will terminate all processes; otherwise it will simply keep track of the match.
     """
 
     # TODO(esteve): This requires internal knowledge about the rmw implementations available.
     def __init__(
         self, name, launch_descriptor, expected_lines, regex_match=False,
-        filtered_prefixes=None
+        filtered_prefixes=None, test_process=True
```

Comment:
What about `primary_process` or `primary_match_handler` to mirror the `primary_exit_handler`? Is that accurate?